docs/source/using/profiles.adoc

```diff
@@ -0,0 +1,35 @@
+= Minishift Profiles
+:icons:
+:toc: macro
+:toc-title:
+:toclevels: 1
+
+toc::[]
+
+
+[[create-multi-instances-of-minishift]]
+== Create multiple instances of Minishift
+
+With the help of profile feature you can create multiple instances of Minishift with xref:../command-ref/minishift_start.adoc#[`minishift start --profile`] command.
```

Comment:
We need some sort of intro introducing the whole concept of profiles. '"Create multiple instances of Minishift" is a single sentence, you read it and go "what just happened". I would think we needs at least one whole paragraph to introduce the idea and what profiles are, before you go `minishift start --profile`.

That said, I think we should do this `minishift --profile foo start`. Profile is a global parameter.